### PR TITLE
Fix issue CODE-3095

### PR DESCRIPTION
**Pull request by javydreamercsw**
Defined inputs and outputs for task. Now it verifies that:
1) The output folder is created (libs in root folder)
2) All sources are present.

Meaning that it will check if one or more of the source files are 
missing and/or the folder doesn't exist altogether before skipping.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -155,7 +155,7 @@ dependencies {
     compile group: 'org.jdom', name: 'jdom2', version:'2.0.6'
     compile group: 'xalan', name: 'xalan', version:'2.7.2'
     compile group: 'xerces', name: 'xercesImpl', version:'2.11.0'
-    compile group: 'xml-apis', name: 'xml-apis', version:'2.0.2'
+    compile group: 'xml-apis', name: 'xml-apis', version:'1.0.b2'
```
Comment (thpr) [inline build.gradle]: This has been addressed elsewhere - needs to be conflict resolved.

Comment (javydreamercsw) [inline build.gradle]: I'll just remove that change and let that solved somewhere else.

**Comment by thpr** [comment]
This needs to be rebased.  We do not merge master into our branches and still use them - it creates a more complicated history.  Please close this PR, create a new local branch, rebase it, then push it to your repo and open a new PR based on that new branch

**Comment by Zaister** [comment]
Oh, sorry, I thought it was a new PR from the message I got from Github. Pleas ignore my comment.

**Pull request closed without merging**